src/acceptance-test/java/org/zalando/nakadi/webservice/SubscriptionAT.java

```diff
@@ -131,9 +132,14 @@ public void testOffsetsCommit() throws Exception {
 
         final Subscription subscription = createSubscriptionForEventType(etName);
 
+        final TestStreamingClient client = TestStreamingClient
+                .create(URL, subscription.getId(), "")
+                .start();
+        Thread.sleep(1000);
```

Comment:
We decided not to use this approach with Thread.sleep if possible, because it's unclean and sometimes can lead to very random behavior. 

Instead you can do something like:
`TestUtils.waitFor(() -> assertThat(client.getSessionId(), not(equalTo("UNKNOWN"))));`

So that you will wait for what you really need, not to abstract 1 second.